pkg/services/sqlstore/dashboard.go

```diff
@@ -307,27 +311,31 @@ func makeQueryResult(query *search.FindPersistedDashboardsQuery, res []Dashboard
 	hits := make(map[int64]*search.Hit)
 
 	for _, item := range res {
-		hit, exists := hits[item.Id]
+		hit, exists := hits[item.ID]
 		if !exists {
 			hit = &search.Hit{
-				Id:          item.Id,
-				Uid:         item.Uid,
+				ID:          item.ID,
+				UID:         item.UID,
 				Title:       item.Title,
-				Uri:         "db/" + item.Slug,
-				Url:         models.GetDashboardFolderUrl(item.IsFolder, item.Uid, item.Slug),
+				URI:         "db/" + item.Slug,
+				URL:         models.GetDashboardFolderUrl(item.IsFolder, item.UID, item.Slug),
 				Type:        getHitType(item),
-				FolderId:    item.FolderId,
-				FolderUid:   item.FolderUid,
+				FolderID:    item.FolderID,
+				FolderUID:   item.FolderUID,
 				FolderTitle: item.FolderTitle,
 				Tags:        []string{},
 			}
 
-			if item.FolderId > 0 {
-				hit.FolderUrl = models.GetFolderUrl(item.FolderUid, item.FolderSlug)
+			if item.FolderID > 0 {
+				hit.FolderURL = models.GetFolderUrl(item.FolderUID, item.FolderSlug)
+			}
+
+			if len(query.Sort.MetaName) > 0 {
```

Comment:
This is the nitpickiest of nitpicks, but for strings specifically I like comparing to the empty string as it tells the reader what type to expect there without having to use a lookup for it.

```suggestion
			if query.Sort.MetaName != "" {
```